applications/conceptual-model-editor/src/action/actions-react-binding.tsx

```diff
@@ -228,6 +265,12 @@ const noOperationActionsContext = {
   addEntitiesFromSemanticModelToVisualModel: noOperation,
   addClassNeighborhoodToVisualModel: noOperation,
   layoutActiveVisualModel: noOperationAsync,
+  //
+  openExtendSelectionDialog: noOperation,
+  openFilterSelectionDialog: noOperation,
+  // TODO PRQuestion: How to define this - Should actions return values?, shouldn't it be just function defined in utils?
+  extendSelection: async () => ({nodeSelection: [], edgeSelection: []}),
+  filterSelection: () => ({nodeSelection: [], edgeSelection: []}),
```

Comment:
The functions are just utility functions extend/filter given set of nodes, so I am not sure if it is action, but it needs a lot contexts to be called.